src/connectors/gcp/pubsub.rs

```diff
@@ -35,7 +35,7 @@ pub(crate) async fn send_message(
         attributes: HashMap::new(),
         message_id: "".into(),
         publish_time: None,
-        ordering_key: "".to_string(),
+        ordering_key: "order".to_string(),
```

Comment:
Should we really hardcode this? Won't that enforce the user into a specific schema?

Reply:
According to the docs, `ordering_key` -  ```If non-empty, identifies related messages for which publish order should be respected. If a Subscription has enable_message_ordering set to true, messages published with the same non-empty ordering_key value will be delivered to subscribers in the order in which they are received by the Pub/Sub system. All PubsubMessages published in a given PublishRequest must specify the same ordering_key value.```

So I decided to keep it a non-empty string for the case when we want to enable message ordering such that all the messages we send will have the same ordering_key that's been hardcoded.

Reply:
Ja it would probably be nice to let the user decide if they need ordering and for which messages, perhaps a `$gpubsub.ordering_key` ?

Reply:
Okay yes sounds good